templates/branding/openSUSE/external_reporting.html.ep

```diff
@@ -7,16 +7,18 @@
 <% my %distri_to_prod = (
     sle => 'SUSE Linux Enterprise',
     opensuse => 'openSUSE',
-);%>
+); %>
 <% my %flavor_to_prod_sle = (
     Server => 'Server',
     Desktop => 'Desktop',
     SAP => 'for SAP Applications',
-);
-%>
+    'Server-HA' => 'High Availability Extension',
+); %>
 % my $distri = $distri_to_prod{$job->DISTRI};
 % my $product = '';
 % if ($job->DISTRI eq 'sle') {
+%     my $flavor = $job->FLAVOR;
+%     $flavor =~ s/Server-DVD-HA/Server-HA-DVD/;
```

Comment:
I don't understand what are you doing here. You initialize `$flavor` from `$job->FLAVOR;` do substitution on `$flavor` but then use `$job->FLAVOR` and `$flavor` is not used anywhere else. Did I miss something?